docs/mllib-clustering.md

```diff
@@ -63,6 +63,10 @@ val clusters = KMeans.train(parsedData, numClusters, numIterations)
 val WSSSE = clusters.computeCost(parsedData)
 println("Within Set Sum of Squared Errors = " + WSSSE)
 {% endhighlight %}
+
+// Save and load model
```

Comment:
Oops!  I missed this before.  This should go before the endhighlight.

Also, you now need to import KMeansModel